scripts/update.sh

```diff
@@ -19,7 +19,7 @@ function copy_daml {
     done
 }
 
-pkgs=$(ls -I "dars*" -I "splitwell*" $CN_REPO_ROOT/daml)
+pkgs=$(ls $CN_REPO_ROOT/daml | grep -v "dars*" | grep -v "splitwell*")
```

Comment:
grep takes regular expressions (BRE by default), so `dars*` would match `dar`, `dars`, `darss` and so on